Src/LiquidProjections.PollingEventStore/PollingEventStoreAdapter.cs

```diff
@@ -186,32 +185,9 @@ private void StartPreloadingNextPage(long precedingCheckpoint, string subscripti
                     throw new OperationCanceledException();
                 }
 
-                CheckpointRequestTimestamp effectiveLastExistingCheckpointRequest =

```

Comment:
Class CheckpointRequestTimestamp is not used anymore, you can remove it

Reply:
In fact, this class was never instantiated before, so that means the old code would never wait, but always continue with `TryLoadNextPageSequentiallyOrWaitForCurrentLoadingToFinish` immediately. That's the actual cause of the issue.

Reply:
> In fact, this class was never instantiated before

It was. There is even a spec that verified that.